thirdparty/nerfstudio/colmap_utils.py

```diff
@@ -0,0 +1,438 @@
+"""Source: Copied from Nerfstudio.
```

Comment:
nit: "forked" from Nerfstudio, which itself forks from COLMAP :)

Reply:
Do we actually need this file? I thought this file was already checked in